packages/modelviewer.dev/examples/tester.html

```diff
@@ -0,0 +1,227 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>&lt;model-viewer&gt; Interactive Example</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, user-scalable=no,
+  minimum-scale=1.0, maximum-scale=1.0">
+  <link type="text/css" href="../styles/examples.css" rel="stylesheet" />
+  <link rel="shortcut icon" type="image/png" href="../shared-assets/models/favicon.png"/>
+  <script src="../node_modules/resize-observer-polyfill/dist/ResizeObserver.js"></script>
+
+  <!-- The following libraries and polyfills are recommended to maximize browser support -->
+
+  <!-- 🚨 REQUIRED: Web Components polyfill to support Edge and Firefox < 63 -->
+  <script src="../node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+
+  <!-- 💁 OPTIONAL: Intersection Observer polyfill for better performance in Safari and IE11 -->
+  <script src="../node_modules/intersection-observer/intersection-observer.js"></script>
+
+  <!-- 💁 OPTIONAL: Resize Observer polyfill improves resize behavior in non-Chrome browsers -->
+  <script src="../node_modules/resize-observer-polyfill/dist/ResizeObserver.js"></script>
+
+  <!-- 💁 OPTIONAL: Fullscreen polyfill is needed to fully support AR features -->
+  <script src="../node_modules/fullscreen-polyfill/dist/fullscreen.polyfill.js"></script>
+
+  <!-- 💁 OPTIONAL: The :focus-visible polyfill removes the focus ring for some input types -->
+  <script src="../node_modules/focus-visible/dist/focus-visible.js" defer></script>
+
+  <!-- 💁 OPTIONAL: Include prismatic.js for Magic Leap support -->
+  <!--<script src="../node_modules/@magicleap/prismatic/prismatic.min.js"></script>-->
+  <style>
+    body, html {
+      height: 100%;
+      font-size: 10pt;
+    }
+
+    label {
+      display: inline-block;
+      width: 120px;
+      height: 24px;
+    }
+
+    input {
+      vertical-align: middle;
+    }
+
+    input.number {
+      border: 0;
+      background: transparent;
+      font-size: 16px;
+      width: 32px;
+      outline: none;
+    }
+
+    select {
+      font-size: 14px;
+    }
+
+    #container {
+      display: flex;
+      height: 100%;
+    }
+
+    model-viewer {
+      flex-grow: 1;
+      height: 100%;
+    }
+
+    #instructions {
+      width: 320px;
+      margin-top: 10px;
+      margin-left: 30px;
+      margin-bottom: 20px;
+    }
+
+    /* Nokia 8810 */
+
+    @media screen and (max-width: 320px) {
+      #instructions {
+        display: none;
+      }
+    }
+  </style>
+</head>
+<body>
+
+  <div id="container">
+    <div id="instructions">
+    <h1>
+      <a href="https://github.com/GoogleWebComponents/model-viewer" target="_blank">&lt;model-viewer&gt;</a>
+    </h1>
+    Drop a <a href="https://github.com/KhronosGroup/glTF" target="_blank">GLB or zipped GLTF</a> for the model =><br/>
```

Comment:
Sounds like we also support unzipped glTFs, right?

Also: prefer glTF/glB over GLTF/GLB.

Reply:
nit: glTF/GLB. It is... not the most memorable. 😅